### PR TITLE
Access Token Proof-of-Possession for msal-browser

### PR DESCRIPTION
**Pull request by pkanher617**
This PR merges all of the previous PRs for access token proof-of-possession to the dev branch to prepare for release tomorrow.

The following PRs are included:

- #2151 Adding req_cnf to the request
- #2153 Handle proof-of-possession tokens correctly
- #2154 Browser implementation of key signing and storage
- #2209 Renewal of proof-of-possession tokens

### CODE REVIEW

File: lib/msal-browser/src/crypto/CryptoOps.ts [hunk 1]
```diff
@@ -60,4 +81,53 @@ export class CryptoOps implements ICrypto {
...
+     * @param resourceRequestUri 
+     */
+    async getPublicKeyThumbprint(resourceRequestMethod: string, resourceRequestUri: string): Promise<string> {
+        const keyPair = await this.browserCrypto.generateKeyPair(CryptoOps.EXTRACTABLE, CryptoOps.POP_KEY_USAGES);
+        const publicKeyJwk: JsonWebKey = await this.browserCrypto.exportJwk(keyPair.publicKey);
+        const privateKeyJwk: JsonWebKey = await this.browserCrypto.exportJwk(keyPair.privateKey);
+        const publicJwkString: string = BrowserCrypto.getJwkString(publicKeyJwk);
+        const publicJwkBuffer: ArrayBuffer = await this.browserCrypto.sha256Digest(publicJwkString);
+        const publicJwkDigest: string = this.b64Encode.urlEncodeArr(new Uint8Array(publicJwkBuffer));
+        const unextractablePrivateKey: CryptoKey = await this.browserCrypto.importJwk(privateKeyJwk, false, ["sign"]);
```
Comment (jasonnutter) [inline lib/msal-browser/src/crypto/CryptoOps.ts:97]: Can you add a comment here about why this key is being reimported?

**Review by jmckennon — APPROVED** [review]
I am assuming that there aren't any other changes besides the ones included from the linked PRs which were already approved, so LGTM!

File: lib/msal-browser/src/crypto/CryptoOps.ts (hunk 1 above)
Comment (pkanher617) [reply lib/msal-browser/src/crypto/CryptoOps.ts:105]: done.

**Comment by pkanher617** [comment]
> I am assuming that there aren't any other changes besides the ones included from the linked PRs which were already approved, so LGTM!

Just bug fixes and sample updates.

**Review by tnorling — APPROVED** [review]

**Review by jasonnutter — APPROVED** [review]

**Review by sameerag — APPROVED** [review]

**Pull request merged**